.circleci/config.yml

```diff
@@ -235,9 +235,9 @@ commands:
               cat <<parameters.extra-requirements-file>> >> /tmp/cachefile
             fi
 
-      - restore_cache:
```

Comment:
Why do you need to change this?

Reply:
it was just testing in a temporary commit. should be gone now